standard/interfaces.md

```diff
@@ -289,9 +411,9 @@ These rules ensure that any covariant or contravariant usage of the interface re
 > <!-- FIX: need to define I<T>, then perhaps break into 2 files, first one a standalone-lib, the second, code-in-class-lib. -->
 > ```csharp
 > class B {}
-> class D : B {}
-> class E : B {}
-> class C : I<D>
+> class D: B {}
+> class E: B {}
+> class C: I<D>
```

Comment:
The typical style is `ClassName : InterfaceName`, so I don't love this change. Elsewhere in the file, new code has been added that similarly does not follow the typical style.